{
  "errors": [],
  "generated_at": "2021-01-13T12:47:28Z",
  "metrics": {
    "/src/test.py": {
      "CONFIDENCE.HIGH": 0,
      "CONFIDENCE.LOW": 0,
      "CONFIDENCE.MEDIUM": 0,
      "CONFIDENCE.UNDEFINED": 0,
      "SEVERITY.HIGH": 0,
      "SEVERITY.LOW": 0,
      "SEVERITY.MEDIUM": 0,
      "SEVERITY.UNDEFINED": 0,
      "loc": 23,
      "nosec": 0
    },
    "_totals": {
      "CONFIDENCE.HIGH": 0,
      "CONFIDENCE.LOW": 0,
      "CONFIDENCE.MEDIUM": 2,
      "CONFIDENCE.UNDEFINED": 0,
      "SEVERITY.HIGH": 1,
      "SEVERITY.LOW": 0,
      "SEVERITY.MEDIUM": 1,
      "SEVERITY.UNDEFINED": 0,
      "loc": 64,
      "nosec": 0
    }
  },
  "results": [
    {
      "code": "50 if __name__ == \"__main__\":\n51     app.run(\"0.0.0.0\",debug=True)\n",
      "filename": "/src/test.py",
      "issue_confidence": "MEDIUM",
      "issue_severity": "HIGH",
      "issue_text": "A Flask app appears to be run with debug=True, which exposes the Werkzeug debugger and allows the execution of arbitrary code.",
      "line_number": 51,
      "line_range": [
        51
      ],
      "test_id": "B201",
      "test_name": "flask_debug_true"
    }
  ]
}
